de.j0chn.nextcloud_password_client.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>de.j0chn.nextcloud_password_client</id>
```

Comment:
`j0chn.de` doesn't seem to exist.

app-id must be a domain or namespace you control. In that case `com.gitlab.j0chn`